waku/v2/node/start_network2.nim

```diff
@@ -9,10 +12,11 @@ import
 import strutils except fromHex
 
 const
-  defaults ="--log-level:TRACE --metrics-logging --metrics-server --rpc"
+  defaults ="--log-level:TRACE --metrics-logging --metrics-server --rpc --discv5-discovery --discv5-table-ip-limit=1024 --discv5-bucket-ip-limit=16 --discv5-bits-per-hop=1"
```

Comment:
Does this mean the nodes spawned by `start_network2` will now always support and run `discv5`?

Reply:
For this experiment yes. I did not try to make the testnet more general so far.
Should I add a setup with just a subset of the nodes supporting discv5, or  should I generalize the whole setup?
I was planning to add `mobile nodes`  later once we agree on a frist synchronization protocol for `mobile nodes`.